README.md

```diff
@@ -109,7 +108,7 @@ The content can be accessed through each specific README link below.
         - Dockerfile
         - AWS ECS
 - [Class 04](classes/04class/README.md)
```

Comment:
Can you add in Class 4 the `Project #1 Kick-off` and in Class 6 `Project #2 kick-off`?